lib/web/apiserver_test.go

```diff
@@ -5758,11 +5763,20 @@ func (s *WebSuite) makeTerminal(t *testing.T, pack *authPack, opts ...terminalOp
 
 	ws, resp, err := dialer.Dial(u.String(), header)
 	if err != nil {
-		return nil, trace.Wrap(err)
+		return nil, nil, trace.Wrap(err)
 	}
 
+	ty, raw, err := ws.ReadMessage()
+	require.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```

Here and below.

Reply:
done

Reply:
It turns out that other tests in this suite rely on the error message emitted from `require.Nil` and fail with this change.

Reply:
I've updated these to return an error instead of asserting on one.